internal/output/emoji.go

```diff
@@ -3,5 +3,6 @@ package output
 // Standard emoji for use in output.
 const (
 	EmojiFailure = "❌"
+	EmojiWarning = "⚠️"
```

Comment:
This triangle renders weirdly in chromium browsers, which includes VSCode and it's terminal.. But what can we do about it :| Just mentioning it because I ran into it before and looked why I keep pasting the wrong triangle in my editor. I can't find the issue anymore, though.

Chrome
![image](https://user-images.githubusercontent.com/19534377/95758529-e622a000-0ca8-11eb-95f8-ffeafca0b451.png)
iTerm (not chromium based)
![image](https://user-images.githubusercontent.com/19534377/95758633-02264180-0ca9-11eb-900d-47fe9f5367a6.png)
vscode terminal (chromium based)
![image](https://user-images.githubusercontent.com/19534377/95758761-241fc400-0ca9-11eb-961d-9839ed25aa92.png)

Reply:
How about ❗️ instead?

Reply:
Hmm, the problem might be 3-byte vs. 6-byte emoji?

Reply:
That might be the case! The❗️works at least.

Reply:
Okay, switched to ❗️, that works in VS Code and here in Chrome. The emoji work in mysterious ways, I guess.